docs/install/install.rst

```diff
@@ -1,51 +1,68 @@
 .. meta::
-  :description: TransferBench documentation 
-  :keywords: TransferBench, API, ROCm, HIP
+  :description: TransferBench is a utility to benchmark simultaneous transfers between user-specified devices (CPUs or GPUs)
+  :keywords: Build TransferBench, Install TransferBench, API, ROCm, HIP
+
+.. _install-transferbench:
 
 ---------------------------
-TransferBench installation
+Installing TransferBench
 ---------------------------
 
-The following software is required to install TransferBench:
+This document provides information required to install and build TransferBench.
```

Comment:
```suggestion
This topic describes how to install and build TransferBench.
```

Reply:
is it build and install or the other way around?

Reply:
Done.